libioc/Config/Jail/BaseConfig.py

```diff
@@ -142,7 +170,8 @@ def clone(
         new_data = libioc.Config.Data.Data(data)
         if current_id is not None:
             new_data["id"] = current_id
-        self.data = new_data
+        for key, value in new_data.items():
+            self.data[key] = value
```

Comment:
no recursive?

Reply:
new_data is a flattened structure. `libioc.Config.Data.Data` handles the translation of keys notated with `.` to a nested structure of dicts.